src/components/TooltipIcon/TooltipIcon.tsx

```diff
@@ -22,9 +22,15 @@ export enum TooltipIconTriggerStyle {
 }
 
 const tooltipTriggerStyleClass: Record<TooltipIconTriggerStyle, string> = {
-    [TooltipIconTriggerStyle.Danger]: 'tw-text-box-negative-strong hover:tw-text-box-negative-strong-hover',
-    [TooltipIconTriggerStyle.Warning]: 'tw-text-box-warning-strong hover:tw-text-box-warning-strong-hover',
-    [TooltipIconTriggerStyle.Primary]: 'tw-text-text-weak hover:tw-text-text-x-weak',
+    [TooltipIconTriggerStyle.Danger]: 'tw-text-box-negative-strong',
```

Comment:
Is it correct to use `box-*` tokens other than the `inverse` ones for text?

Reply:
You're right. I've updated these with the inverse versions.